lib/apis/v3/policy.go

```diff
@@ -57,6 +57,15 @@ type Rule struct {
 	Source EntityRule `json:"source,omitempty" validate:"omitempty"`
 	// Destination contains the match criteria that apply to destination entity.
 	Destination EntityRule `json:"destination,omitempty" validate:"omitempty"`
+
+	// HTTP contains match criteria that apply to HTTP requests.
+	HTTP HTTPRule `json:"http,omitempty" validate:"omitempty"`
```

Comment:
Should this be a struct pointer?   I think you'll get "http: {}" output otherwise - even if this feature is not enabled.

Reply:
@robbrockbank good point. I've made the change. thx